portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,61 @@
+package com.google.sps.data;
+
+import java.util.Date;
+import com.google.appengine.api.datastore.Entity;
+
+
+/** Class containing users comments and time. */
+public final class Comment {
+
+  private final String username;
```

Comment:
The "final" is redundant here, I think. This class, the way you've written it, is immutable since it provides no way to modify the instance variables so there would be no way to reassign them, even if you wanted. So you can drop the "final" part.

Reply:
still unresolved

Reply:
I kind of like declaring final here, it makes it clear upfront that these are immutable and would prevent someone from accidentally adding Set() methods. But I'm ok either way.